src/components/MessageSender.js

```diff
@@ -1,7 +1,74 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 
-export default function MessageSender() {
-	return <Wrapper>메시지 전송부</Wrapper>;
+export default function MessageSender({ addToList }) {
+  const [message, setMessage] = useState('');
+
+  const handleChangeMessage = (event) => {
+    setMessage(event.target.value);
+  };
+
+  const handleSubmitMessage = (event) => {
+    event.preventDefault();
+
+    // 입력이 없을 때 예외 처리
+    !message && alert('메세지를 입력해주세요!');
+    message && addToList(message);
```

Comment:
조건문대신 논리연산자로 처리하신부분 간결하고 특이하네요👍

Reply:
간단한 처리 하나 배워갑니다!! 혹시 이런식으로 간추려서도 될까 싶어 의견 제시해 봅니다!
```suggestion
    !message ?  alert('메세지를 입력해주세요!') : addToList(message);
```